src/browser/api/window.js

```diff
@@ -1362,7 +1362,7 @@ Window.showAt = function(identity, left, top, force = false) {
     let defaultAction = () => {
         let currentBounds = browserWindow.getBounds();
 
-        browserWindow.setBounds({
+        browserWindow.setBounds({ // no need to call clipBounds as width and height not changing
```

Comment:
Sounds like an extra/unnecessary comment to me.
But if we keep it, probably should go onto a separate line.